tests/unit/ui/components/TestUserAvatar.js

```diff
@@ -37,13 +37,13 @@ describe(__filename, () => {
     const user = getCurrentUser(state.users);
     const root = renderUserAvatar({ className: 'test', user });
 
-    expect(root.find('img.test')).toHaveLength(1);
+    expect(root.find('.UserAvatar.test')).toHaveLength(1);
   });
 
   it('renders extra classNames when rendering an Icon component', () => {
```

Comment:
Actually this test can be removed entirely as we are rendering the className in the root `<div>` now and it isn't contextual based on which component we're outputting.